src/authenticationinapp/authenticationinapp.h

```diff
@@ -135,6 +139,8 @@ class AuthenticationInApp final : public QObject {
  private:
   State m_state = StateInitializing;
 
+  int m_verificationCodeLength = 6;
```

Comment:
This seems super static, right? Don't make it a variable. Do this:
1. add Q_PROPERTY as written above
2. in the .h write only: `static int verificationCodeLength() { return 6; }`